app/views/organisations/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Departments, agencies and public bodies - GOV.UK" %>
+
+<h1>Departments, agencies and public bodies - GOV.UK</h1>
+<br>
+<h2>Ministerial departments</h2>
+<%= @content.select { |item| item[:department_type] == "Ministerial" }.count %>
+<ul>
+  <% @content.each do |item| %>
+    <% if item[:department_type] == "Ministerial" %>
```

Comment:
it seems you need two methods to filter out "Ministerial" and "Non Ministerial".
There's a few things here that are problematic: 

- There's a lot of repetition in this view
- You are cycling through the whole `@content` twice which adds complexity
- There's a lot of logic in this view that should belong somewhere else, for example: a view presenter, or a ContentItemHelper with the logic seems a more appropriate place to send this.